R/search.R

```diff
@@ -48,7 +48,8 @@ enumerate_primers = function(forward, reverse){
   forward_primers = enumerate_ambiguity(forward)
   data.frame(forward=forward_primers,
              reverse=rep(enumerate_ambiguity(reverse),
-                         each=length(forward_primers)))
+                         each=length(forward_primers)),
+             stringsAsFactors = F)
```

Comment:
This should really be `stringsAsFactors = FALSE`. `F` is just a variable that happens to be set to `FALSE`, it can actually be assigned any value. `FALSE` is a keyword which cannot be assigned.

``` r
F <- TRUE
if (F) print("hi")
#> [1] "hi"
FALSE <- TRUE
#> Error in FALSE <- TRUE: invalid (do_set) left-hand side to assignment
```

Created on 2018-01-17 by the [reprex package](http://reprex.tidyverse.org) (v0.1.1.9000).

Reply:
Good to know, I'll fix that.